CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing Guideline 
+
+1. Use naming format ```<judge> <problem id> - <problem name> <version>```. For example -
+
```

Comment:
You can mention that, version number only for >2, no v1.

Reply:
Done